docs/bugs-features.rst

```diff
@@ -0,0 +1,155 @@
+is ok if you aren't able to provide all the information requested here, but clear and detailed
+issues are more likely to be fixed quickly. Bonus points if they are `pretty
+<https://www.redmine.org/projects/redmine/wiki/RedmineTextFormattingMarkdown>`_.
+
+For **Issues** (Bugs) please include:
+
+#. Detailed explanation of the problem. For problems involving external content sources, please
+   indicate the source (and a link) if you can.
+#. Clear steps to reproduce the problem. Commands and/or REST calls are highly encouraged.
+#. Expected results
+#. Actual results
+#. Snippet of relevant logs, especially Exceptions.
+
+You can also upload attachments, but please only upload relevant data. For example, if you have an
+entire log which contains some errors, please trim it to just the relevant portions and upload
+those.
+
+For **Feature Requests** (Stories), the description will depend on the feature. Please be specific
+when describing the requested behavior and include the motivation for adding it. If you have
+suggestions for how the commands/REST calls would look, please include that as well. Feature
+requests require follow-up from the filer, so please :ref:`reach out<community>` with a link to
+your issue.
+
+.. _triage:
+
+Triage
+------
+Twice per week, the Pulp team triages all new bugs, at which point its *Severity* rating and other
+aspects of the report will be evaluated. If necessary, the bug may be commented on requesting more
+information or clarification from the reporter. When a bug has enough information, its *Priority*
+rating set and is marked as triaged using the *Triaged* boolean.
+
+
+.. _security-bugs:
+
+Security Disclosures
+--------------------
+
+We take security issues seriously and welcome responsible disclosure of security vulnerabilities in
+Pulp. Please email pulp-security@redhat.com (a private address for the Pulp Security Team) with all
+reports.
+
+Your report should include:
+
+#. Pulp version
+#. A vulnerability description
+#. Reproduction steps
+#. Feel free to submit a patch with your disclosure. A member of the Pulp Security Team will
+   confirm the vulnerability, determine its impact, and develop a fix.
+
+.. _redmine-fields:
+
+Redmine Fields
+--------------
+
++-------------+-----------------------------------------------------------------------------------+
+| Field       | Description                                                                       |
++-------------+-----------------------------------------------------------------------------------+
+| Tracker     | - ``Issue`` (bug) Defect in a feature that is expected to work.                   |
+|             | - ``Story`` New feature or functionality.                                         |
+|             | - ``Refactor`` Improvement that will not be visible to the user in any way.       |
+|             | - ``Task`` Infrastructure work that will not be a part of released code.          |
```

Comment:
I don't think a task ought to be limited to infrastructure work. Also, I'm kind of partial to the definitions we came up with on https://pulp.plan.io/issues/3447.

Reply:
@daviddavis, this was derived from 3447. Hows " Work that will not be part of released code."

> Work that doesn't require a release. Documentation would not be a task since it has to be released whereas blog post would be a task because it's not part of any Pulp release.

"require a release" doesn't seem quite right. The documentation part seemed explanatory, not part of the definition.

Reply:
That works. I might address 3447 after you get this merged because I'd like to provide some examples, etc (unless you plan to do that already).

Reply:
My main goal was to get something up with a good structure. Im happy to make those changes if you prefer, but it might be easier to just get this merged and you can make those changes in a separate PR.

Reply:
Actually I think this is fine for now. +1 to just updating the definition for Task.